security/custom_authentication_provider.rst

```diff
@@ -4,655 +4,14 @@
 How to Create a custom Authentication Provider
 ==============================================
 
-.. tip::
+Creating a custom authentication system is hard, and this entry will walk
```

Comment:
I agree with this, but I'd like to improve the introduction of this article. This is not enough for me --> *"Creating a custom authentication system is hard..."*  I'd like to know (very briefly!!) which is exactly a "custom authentication system" and in which edge cases should I consider it. In other words: what am I trying to do that Symfony can't provide it for me. Thanks!